app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -59,7 +59,9 @@ export const FixedSessionsMapCtrl = (
 
     const defaults = {
       sensorId,
-      location: {address: "", indoorOnly: false, streaming: true},
+      location: '',
+      indoorOnly: false,
+      streaming: true,
```

Comment:
of course, this breaks the checkboxes for now, we need are planning to rewrite them next.